test/index.test.ts

```diff
@@ -49,6 +49,16 @@ describe('monika', () => {
     getPublicNetworkInfoStub.restore()
   })
 
+  test
```

Comment:
Maybe we should test the case where the config is loading too. Use this gist: `https://gist.github.com/dennypradipta/8193f47a6c23ecd263a24b68e69e4494`

Reply:
done.
![image](https://user-images.githubusercontent.com/964106/218666095-e556face-a777-4220-94cf-3fbb1a828c4f.png)